### DISCUSSION TITLE
bind -n M-Right next-window not working in tmux.

### DISCUSSION THREAD

**Discussion by fibbo** [ghostty-org/ghostty#3951] [Q&A] [answered]
My problem might be related to https://github.com/ghostty-org/ghostty/discussions/3447, but I'm not sure.

I have 

```
bind -n M-Right next-window
bind -n M-Left previous-window
```

in my `.tmux.conf` to switch between windows. This worked under alacritty but in ghostty it doesn't work. If the tmux window has nvim open, it opens `harpoon` (which is bound to `ctrl-e`).

**Comment by 00-kat** [comment, marked as answer]
Check if they are bound in Ghostty (I think they are by default).  `ghostty +list-keybinds`.

If they are,
```ini
keybind = alt+right=unbind
keybind = alt+left=unbind
```

**Reply by fibbo** [reply]
Thank you! After unbinding the keybind, it worked. 

It is a strange keybind though:
```
# keybind = super + right=text:\x05
# keybind = super + left=text:\x01
```

What is the purpose of these keybindings?
